dz2.1/ДЗ. TypeScript. Анотація типів та примітивні типи.ts

```diff
@@ -0,0 +1,116 @@
+class School {
+  directions: Direction[];
+
+  addDirection(direction: Direction) {
+    this.directions.push(direction);
+  }
+}
+
+class Direction {
+  levels: Level[];
+
+  get name() {
+    return this._name;
+  }
+
+  constructor(name: string) {
+    this._name = name;
+  }
+
+  addLevel(level: Level) {
+    this.levels.push(level);
+  }
+}
+
+class Level {
+  groups: Group[];
+
+  constructor(name: string, program: string) {
+    this.name = name;
+    this._program = program;
+  }
+
+  get name(): string {
+    return this._name;
+  }
+
+  get program(): string {
+    return this._program;
+  }
+
+  addGroup(group: Group) {
```

Comment:
Please, do not forget to explicitly add type annotation for the return function value.